src/main/java/com/parasoft/findings/sonar/MapperUtil.java

```diff
@@ -57,9 +59,17 @@ public static SoftwareQuality mapToSonarImpactSoftwareQuality(String category, b
 
     public static CleanCodeAttribute mapToSonarCleanCodeAttribute(String category) {
         String tag = categoryToTag(category);
-        // TODO will be implemented in CICD-666
-        // The following return value is just for testing, because the default value is CleanCodeAttribute.CONVENTIONAL if not set.
-        return CleanCodeAttribute.CLEAR;
+        if (tag.contains(".")) {
+            tag = tag.substring(0, tag.indexOf("."));
+        }
+        if (TagPrefixMap.TAG_PREFIX_MAP.containsKey(tag)) {
+            return TagPrefixMap.TAG_PREFIX_MAP.get(tag);
+        }
+        // codsta, bean, xml, dbc, ejb, eclipse, javadoc, spring, servlet, struts, port, inter, props, hibernate,
```

Comment:
Please add some comments here to explain that we'll classify the listed categories to CONVENTIONAL. And we'll also classify unknown categories into CONVENTIONAL as default.